ferrocene/ignored-tests.toml

```
@@ -58,3 +58,23 @@ reason = "QEMU user space emulation uses a different exit code for aborting"
 tests = ["tests/codegen/vec_pop_push_noop.rs"]
 targets = ["aarch64-unknown-ferrocenecoretest"]
 reason = "Remove when https://github.com/rust-lang/rust/issues/127395 is fixed"
+
+[["tests/ui"]]
+tests = [
+    "tests/ui/backtrace/dylib-dep.rs",
+    "tests/ui/backtrace/line-tables-only.rs",
+    "tests/ui/backtrace/std-backtrace.rs",
+    "tests/ui/backtrace/backtrace.rs",
+    "tests/ui/ferrocene/compiler-arguments/print/print_target-libdir.rs",
+    "tests/ui/panics/issue-47429-short-backtraces.rs#legacy",
+    "tests/ui/panics/issue-47429-short-backtraces.rs#v0",
+    "tests/ui/panics/runtime-switch.rs#legacy",
+    "tests/ui/panics/runtime-switch.rs#v0",
+    "tests/ui/panics/short-ice-remove-middle-frames-2.rs",
+    "tests/ui/panics/short-ice-remove-middle-frames.rs",
+    "tests/ui/runtime/backtrace-debuginfo.rs",
+    "tests/ui/structs-enums/enum-rec/issue-17431-6.rs",
+    "tests/ui/threads-sendsync/tcp-stress.rs",
+]
+targets = ["x86_64-pc-nto-qnx710"]
+reason = "To be fixed in follow-up PRs"
```

```suggestion
reason = "Experimental target which does not yet pass all tests. To be fixed in follow-up PRs"
```

I'm not sure where this is shown, but let's remind ourselves why it's OK to do this.